src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -213,7 +213,9 @@ export default ({
         const props: MouseEvent = {
           x: position?.x,
           y: position?.y,
-          ...(position ? getLocationFromScreenXY(viewer.scene, position.x, position.y) ?? {} : {}),
+          ...(position
+            ? getLocationFromScreenXY(viewer.scene, position.x, position.y, true) ?? {}
```

Comment:
Why are you setting withTerrain to true here on all mouse events with position? If you hardcode that, is there a reason to having the withTerrain prop?

Reply:
This is the difference between the interest that the plug-in has and the interest that the map engine has. Plug-ins only get the basic terrain-enabled location, but there may be times when you want to do pure location acquisition inside Re:Earth, independent of the plug-in. In that case, you are still calling a function exposed by the map engine. There is no reason to go to the trouble of making changes that narrow the degree of freedom.